src/helpers/schema.sql

```diff
@@ -58,6 +58,9 @@ CREATE TABLE proposals (
   scores_state VARCHAR(24) NOT NULL,
   scores_total DECIMAL(64,30) NOT NULL,
   scores_updated INT(11) NOT NULL,
+  vp_usd decimal(64,30) NOT NULL DEFAULT '0.000000000000000000000000000000',
+  vp_eth decimal(64,30) NOT NULL DEFAULT '0.000000000000000000000000000000',
+  vp_value_by_strategy json NOT NULL,
```

Comment:
These cols are only used internally, not used in our backend/testnet. so not sure if we want everyone to use these to hold default values

@bonustrack wyt?

Reply:
Whats wrong about default value?